docs/docsite/rst/installation_guide/installation_distros.rst

```diff
@@ -7,31 +7,35 @@ Installing Ansible on specific operating systems
 
 The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.
 
-The following instructions will guide you through installing the ``ansible`` package with your preferred distribution's package manager.
+This document guides you through installing the ``ansible`` package with different package managers.
```

Comment:
```suggestion
This document guides you through installing Ansible from different distribution's package repositories.
```

or

```suggestion
This document guides you through installing Ansible from your preferred distribution's package repositories.
```

- "Ansible" instead of " `ansible`", as the instructions include core and collections as well
- "different package managers" isn't entirely correct. The docs are talking about specific distributions. Two distributions may use the same package manager, but that doesn't mean they both have the ansible package.

Reply:
sounds fair, thanks